content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -470,7 +471,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `RunAsGroup`: Enable control over the primary group ID set on the init processes of containers.
 - `RuntimeClass`: Enable the [RuntimeClass](/docs/concepts/containers/runtime-class/) feature for selecting container runtime configurations.
 - `ScheduleDaemonSetPods`: Enable DaemonSet Pods to be scheduled by the default scheduler instead of the DaemonSet controller.
-- `SCTPSupport`: Enables the usage of SCTP as `protocol` value in `Service`, `Endpoint`, `NetworkPolicy` and `Pod` definitions
+- `SCTPSupport`: Enables the usage of SCTP as `protocol` value in `Service`, `Endpoint`, `EndpointSlice`, `NetworkPolicy` and `Pod` definitions
```

Comment:
Mostly fixing what was already wrong, but good to clean up:
```suggestion
- `SCTPSupport`: Enables the _SCTP_ `protocol` value in Pod, Service, Endpoints, EndpointSlice, and NetworkPolicy definitions.
```

Note that [Endpoints](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpoints-v1-core) is plural.

Reply:
Fixed.